Doc/glossary.rst

```diff
@@ -754,7 +754,7 @@ Glossary
 
    loader
       An object that loads a module. It must define a method named
-      :meth:`load_module`. A loader is typically returned by a
+      :meth:`!load_module`. A loader is typically returned by a
```

Comment:
Here you could link to https://docs.python.org/3/library/importlib.html#importlib.abc.Loader.load_module

But I think this glossary entry may be outdated, since the docs for that method describe it as being "deprecated since Python 3.4"

Reply:
I tried that change locally, but even after rebuilding the html docs I couldn't see that link. Maybe the deprecation had something to do with that.

Reply:
Not sure, but I still think the "principled" way to fix this reference warning would be to update it so that it doesn't talk about a deprecated method at all anymore. The Sphinx warning is a useful reminder that there's something incorrect here, in the meantime :)

That might be big enough to be its own PR -- maybe this change should just be reverted for now?

Reply:
Can we save that for a different PR? I'm not qualified to describe/define this term. I was just trying to eliminate the reference nit.

Reply:
Sounds good -- could you revert this change for now, then? If we leave the reference warning there, we'll have a reminder that there's something that needs to be fixed here :)